src/main/sources/GA4GHAlignmentSource.js

```diff
@@ -52,6 +41,9 @@ function create(spec: GA4GHSpec): AlignmentDataSource {
   var coveredRanges: ContigInterval<string>[] = [];
 
   function addReadsFromResponse(response: Object) {
+    if (response.alignments === undefined) {
+      return;
```

Comment:
curious why/when this happens. I would have expected the `alignments` to be an empty array for the semantics of it, but is this how `GA4GH` server responds when no reads match the criteria?